### ISSUE TITLE
Co-references between sentences

### ISSUE THREAD

**Issue by david-waterworth** [inception-project/inception#2519]
Is it possible to annotate co-references by selecting two named entities that are the same entity?

For example, I've created an entity layer that is bound to tokens and sentence crossing is not allowed. I've also created a relationship layer bound to these entities.

I now also want to annotate coreference clusters, i.e. the same entity appears in multiple sentences.

One way of doing this seems to be to use the identifier field of the entity layer to give all instances of a named entity that belong to the same cluster the same KB id. 

It would be quicker though to be able to select a cluster, say by shift clicking all occurrences in a cluster. Is something like this possible?

I tried using a coreference / chain layer but I get errors due to sentence crossing.

**Comment by reckart** [comment]
I'm afraid multi-selection is currently not possible.

The coref layer should allow cross-sentence annotation... that's odd.

However, chains cannot be curated and also e.g. agreement calculation is not supported.

**Comment by reckart** [comment]
What you can do is this: go to the feature settings for your KB feature and enable the option "Remember". After that is enabled, when you annotate an entity, link it to a KB item and then annotate another entity, it should be automatically linked to the same KB item.

**Comment by reckart** [comment]
I have create a new project ("span/relation type"), then manually added the pre-defined coref layer in the project settings. The pre-defined layer has "Allow crossing sentence boundaries" turned on by default. I had no problem linking two spans that were both in different sentences.

Actually, even creating a custom chain layer with the "allow crossing sentence boundaries" disabled, I was able to create two spans in two different sentences and linking the.

Turns out that the "Allow crossing sentence boundaries" seems to affect only the spans of chain layers and the chain links are always allowed to cross sentence boundaries....
